.lintstagedrc

```diff
@@ -1,5 +1,5 @@
 {
     "*.js": ["eslint"],
     "*.{css,less}": ["stylelint"],
-    "*.{ts,tsx}": ["npm run lint-ts"]
+    "*.{ts,tsx}": ["eslint --ext .tsx,.ts"]
```

Comment:
```suggestion
    "*.{ts,tsx}": ["eslint"]
```
No need to pass a file extension parameter - you already did this via `"*.{ts,tsx}"`